views/layout.jade

```diff
@@ -9,6 +9,15 @@ html
   body.bg-dark
     nav.clearfix.bg-light.white.sm-show
       div.container
+        //- if messages.error
```

Comment:
This section was to be used with the express-flash package. It's optional. If we don't need it I can remove these lines.